### ISSUE TITLE
Request: Starlight theme

### ISSUE THREAD

**Issue by Seres67** [NvChad/base46#370]
I've been using this theme for a while, i switched to NvChad to test it but i can't figure out a way to make it work.
[Here](https://github.com/CosmicToast/starlight/blob/main/colors/colors.sh) are the colors, [here](https://github.com/Seres67/kickstart.nvim/blob/master/colors/starlight.lua) is what i did for my kickstart config

**Comment by siduck** [comment]
I will make a base46 variant for it later

**Comment by siduck** [comment]
@Seres67  u got any pics of it?

**Comment by Seres67** [comment]
Screenshot of my kickstart config: ![](https://github.com/user-attachments/assets/a3f1d7e6-c870-4361-b0fb-ecb654186c10)

**Comment by siduck** [comment]
switch to v3.0 branch and test your theme

**Comment by Seres67** [comment]
Left is my config, right is NvChad.
![image](https://github.com/user-attachments/assets/a89fb129-8f6a-49f3-a2d8-68c088d6e934)


How do I change branches? I've just added the file manually. Do I just change NvChad branch as a whole?

**Comment by siduck** [comment]
{ base46 blabla , branch='v3.0' }

**Comment by siduck** [comment]
i hope u did tsintall typescript javascript tsx html on nvchad?

**Comment by Seres67** [comment]
yes i did, i've been using nvchad with another theme while waiting

**Comment by siduck** [comment]
![image](https://github.com/user-attachments/assets/26854627-759d-4f39-a9a5-da652739dcc1)

**Comment by Seres67** [comment]
oooo pretty, did you change anything or is it on my end?

**Comment by siduck** [comment]
> oooo pretty, did you change anything or is it on my end?

no, i hope you're on v3.0 branch

**Comment by Seres67** [comment]
got it to work by reinstalling all TS parsers. thank you very much!

EDIT: actually i didn't have html parser installed. weird since i'm working on html files daily.
